lsl_autobids/config_globals.py

```diff
@@ -1,6 +1,33 @@
 import os
 import yaml
 
+class CLIArgs:
+    _instance = None
+
+    def __init__(self):
+        self._args = None
+        self._defaults = {
+            "project_name": "default_project",
+            "yes": False,
+            "redo_bids_conversion": False,
```

Comment:
I wonder if we need a more finegrained decision between redoing nothing prior to the last successful execution, redo everything, or just the files that are missing?